spec/requests/topics_controller_spec.rb

```diff
@@ -1088,35 +1088,18 @@ def extract_post_stream
       end
 
       before do
+        TopicView.stubs(:chunk_size).returns(4)
         @post_ids = topic.posts.pluck(:id)
-        22.times do
+        5.times do
           @post_ids << Fabricate(:post, topic: topic).id
         end
       end
 
-      it 'grabs first page when no filter is provided' do
-        get "/t/#{topic.slug}/#{topic.id}.json"
-        expect(response).to be_success
-        expect(extract_post_stream).to eq(@post_ids[0..19])
-      end
-
-      it 'grabs first page when first page is provided' do
-        get "/t/#{topic.slug}/#{topic.id}.json", params: { page: 1 }
-        expect(response).to be_success
-        expect(extract_post_stream).to eq(@post_ids[0..19])
-      end
-
-      it 'grabs correct range when a page number is provided' do
```

Comment:
@OsamaSayegh I think you misunderstood what @SamSaffron meant. We still need these tests but instead of running them separately in 5 different `it` blocks. We can just group them together under a single `it` block.

Reply:
Oh, clearly I needed some sleep. That's done ✅